salt/fileserver/roots.py

```diff
@@ -130,27 +130,36 @@ def update():
     mtime_map_path = os.path.join(__opts__['cachedir'], 'roots/mtime_map')
     # data to send on event
     data = {'changed': False,
+            'files': {'changed': []},
```

Comment:
Why 'removed' and 'added' are not initialized here to an empty list as well?. Having said that, I'm not even sure is worth initialize the 'files' member at all when 'changed' is False?